<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_editUser</class>
 <widget class="QWidget" name="form_editUser">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>352</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>781</width>
    <height>352</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Редактировать пользователя</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Обязательные поля&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Фамилия:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Имя:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Электронная почта:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Орг. подразделение:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editLastName">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Фамилия&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Максимальное количество символов: 60.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editFirstName">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Имя&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Максимальное количество символов: 60.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editPrimaryEmail">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Электронная почта&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Имя пользователя для входа. Введите адрес полностью: &lt;span style=&quot; font-style:italic;&quot;&gt;imyapolzovatelya@vashdomen.com&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="editPassword">
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Пароль&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Пароли вводятся с учетом регистра. По умолчанию пароль должен содержать не менее 8 символов.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Если вы обновляете данные существующих пользователей и не хотите менять их пароли, введите &lt;span style=&quot; font-weight:600;&quot;&gt;**** &lt;/span&gt;(4 звездочки).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>****</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="editOrgUnitPath">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Организационное подразделение&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Здесь указывается, к какому организационному подразделению относится пользователь.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Если вы ещё не создали организационную иерархию&lt;/span&gt;, введите &lt;span style=&quot; font-weight:600;&quot;&gt;только&lt;/span&gt; косую черту (&lt;span style=&quot; font-weight:600;&quot;&gt;/&lt;/span&gt;), чтобы указать, что все пользователи относятся к &lt;span style=&quot; font-weight:600;&quot;&gt;родительскому организационному подразделению верхнего уровня&lt;/span&gt;. Вы можете создать другие подразделения позже и перенести туда пользователей.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Если вы уже создали организационную иерархию&lt;/span&gt;, используйте показанный в примерах ниже формат, чтобы указать, к какому подразделению относится пользователь (родительского или дочернего уровня).&lt;/p&gt;&lt;p&gt;Примеры:&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;/&lt;/span&gt; (для &lt;span style=&quot; font-weight:600;&quot;&gt;родительского подразделения верхнего уровня не указывайте&lt;/span&gt; доменное имя;&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;/Продажи&lt;/span&gt; (для дочерних подразделений);&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;/Студенты/Первокурсники&lt;/span&gt;.&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="buttonCreatePassword">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Сгенерировать новый пароль</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QFormLayout" name="formLayout_3">
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Резервная эл. почта:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Резервный моб. телефон:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Личный номер сотрудника:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Рабочий адрес:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Домашний адрес:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Новая основная эл. почта:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Принудительная смена пароля:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Статус сотрудника:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>396</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Необязательные поля&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="buttonCreateEmployeeId">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Сгенерировать личный номер</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="editRecoveryEmail">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Резервный адрес электронной почты&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Используйте адрес электронной почты, не входящий в ваш основной и дополнительные домены.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="editRecoveryMobilePhone">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Резервный номер мобильного телефона&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Указывайте номер телефона в формате &lt;span style=&quot; font-weight:600;&quot;&gt;E.164&lt;/span&gt; - сначала плюс (+), затем код страны, код региона и собственно номер. Пример: &lt;span style=&quot; font-weight:600;&quot;&gt;+16505551212&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="inputMask">
          <string>+7 (000) 000 00-00</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="editEmployeeId">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Идентификационный номер сотрудника&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Иногда этот номер нужно указывать при дополнительной аутентификации. Он может содержать цифры, буквы и специальные символы.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="inputMask">
          <string>00000000000000000000000000000000</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="editWorkAddress">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Рабочий адрес&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Если адрес содержит запятые или переносы строки, заключите его в двойные кавычки. Пример: &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;Москва, улица Строителей, д. 25&amp;quot;&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="editHomeAddress">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Домашний адрес&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Если адрес содержит запятые или переносы строки, заключите его в двойные кавычки. Пример: &lt;span style=&quot; font-weight:600;&quot;&gt;&amp;quot;Москва, улица Строителей, д. 25&amp;quot;&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="editNewPrimaryEmail">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Новый основной адрес электронной почты&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Это поле исползуется только для редактирования аккаунтов существующих пользователей. Укажите новый основной адрес электронной почты (имя пользователя).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="comboChangePassword">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Смена пароля при следующем входе&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Чтобы при следующем входе в аккаунт пользователи получили запрос на смену пароля, выберите &amp;quot;&lt;span style=&quot; font-weight:600;&quot;&gt;Да&lt;/span&gt;&amp;quot;. Если этого не требуется, оставьте &amp;quot;&lt;span style=&quot; font-weight:600;&quot;&gt;Нет&lt;/span&gt;&amp;quot;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Нет</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Да</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="comboEmployeeStatus">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Новый статус сотрудника&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Это поле используется только для редактирования аккаунтов &lt;span style=&quot; font-weight:600;&quot;&gt;существующих &lt;/span&gt;пользователей. Чтобы заблокировать пользователя или отправить его в архив, выберите &lt;span style=&quot; font-weight:600;&quot;&gt;Suspended&lt;/span&gt; или &lt;span style=&quot; font-weight:600;&quot;&gt;Archived&lt;/span&gt;. Чтобы восстановить аккаунт пользователя, выберите &lt;span style=&quot; font-weight:600;&quot;&gt;Active&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <item>
          <property name="text">
           <string>Active</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Suspended</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Archived</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Сохранить данные о пользователе</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCreateEmployeeId</sender>
   <signal>clicked()</signal>
   <receiver>form_editUser</receiver>
   <slot>buttonCreateEmployeeId_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>686</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCreatePassword</sender>
   <signal>clicked()</signal>
   <receiver>form_editUser</receiver>
   <slot>buttonCreatePassword_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>form_editUser</receiver>
   <slot>buttonSave_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonCreatePassword_clicked()</slot>
  <slot>buttonCreateEmployeeId_clicked()</slot>
  <slot>buttonSave_clicked()</slot>
 </slots>
</ui>
